### PR TITLE
282 remove usages of unwrap and expect from spacewalk pallets

### PR DESCRIPTION
**Pull request by RustNinja**
update spacewalk's pallet.
replace `unwrap` and `expect` functions with proper handling.
Now there are nothing that will panic in runtime
issue #282

### CODE REVIEW

File: pallets/stellar-relay/src/lib.rs [hunk 1]
```diff
@@ -117,6 +117,7 @@ pub mod pallet {
 		TransactionSetHashCreationFailed,
 		TransactionSetHashMismatch,
 		ValidatorLimitExceeded,
+		FailedToComputenonGenericTxSetContentHash,
```
Comment (ebma) [inline pallets/stellar-relay/src/lib.rs:120]: Nit: The 'n' for non should be capitalized
```
		FailedToComputeNonGenericTxSetContentHash,
```

File: pallets/stellar-relay/src/testing_utils.rs [hunk 2]
```diff
@@ -197,7 +197,7 @@ pub fn build_dummy_proof_for<T: crate::Config>(
 	txes.push(transaction_envelope.clone()).unwrap();
 	let transaction_set = TransactionSet { previous_ledger_hash: Hash::default(), txes };
 
-	let tx_set_hash = crate::compute_non_generic_tx_set_content_hash(&transaction_set);
```
Comment (ebma) [inline pallets/stellar-relay/src/testing_utils.rs:200]: Don't we need to handle this now? You changed it to return an `Option` now, but the `create_scp_envelope` function that consumes it a few lines below still expects the real type and not an option I think.

File: pallets/stellar-relay/src/tests.rs [hunk 3]
```diff
@@ -105,7 +105,7 @@ fn create_valid_dummy_scp_envelopes(
 	txes.push(transaction_envelope.clone()).unwrap();
 	let transaction_set = TransactionSet { previous_ledger_hash: Hash::default(), txes };
 
-	let tx_set_hash = crate::compute_non_generic_tx_set_content_hash(&transaction_set);
```
Comment (ebma) [inline pallets/stellar-relay/src/tests.rs:108]: Same here, I think we need to use a `match` or something

**Review by ebma — COMMENTED** [review]
Maybe I'm wrong about the changes I point out. The tests seem to be working(?)

File: pallets/stellar-relay/src/lib.rs (hunk 1 above)
Comment (cr4pt0) [reply pallets/stellar-relay/src/lib.rs:120]: Done.

File: pallets/stellar-relay/src/testing_utils.rs (hunk 2 above)
Comment (cr4pt0) [reply pallets/stellar-relay/src/testing_utils.rs:200]: I added `unwrap` method but now i replace it with 
`.expect("Should compute non generic tx set content hash");`

File: pallets/stellar-relay/src/tests.rs (hunk 3 above)
Comment (cr4pt0) [reply pallets/stellar-relay/src/tests.rs:108]: I added `unwrap` method but now i replace it with 
`.expect("Should compute non generic tx set content hash");`

**Pull request merged**